src/qml/pages/wallet/RequestPayment.qml

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import org.bitcoincore.qt 1.0
+
+import "../../controls"
+import "../../components"
+import "../settings"
+
+Page {
+    id: root
+    background: null
+
+    property int requestCounter: 0
+
+    ScrollView {
+        clip: true
+        width: parent.width
+        height: parent.height
+        contentWidth: width
+
+        CoreText {
+            id: title
+            anchors.left: contentRow.left
+            anchors.top: parent.top
+            anchors.topMargin: 20
+            text: qsTr("Request a payment")
+            font.pixelSize: 21
+            bold: true
+        }
+
+        RowLayout {
+            id: contentRow
+            anchors.top: title.bottom
+            anchors.topMargin: 40
+            anchors.horizontalCenter: parent.horizontalCenter
+            spacing: 30
+            ColumnLayout {
+                id: columnLayout
+                Layout.minimumWidth: 450
+                Layout.maximumWidth: 470
+
+                spacing: 5
+
+                Item {
```

Comment:
nit: Maybe amount input deserves it's own control?

Reply:
Yes, it will be. I haven't committed to doing that just yet as I don't have a clear idea of exactly how the BitcoinAmounts and the Bitcoin Address inputs will be handled. I think the PR to do the input formatting of the amounts should take care of having this component extracted out with clear signals/properties.